bundles/org.openhab.binding.mystrom/src/main/java/org/openhab/binding/mystrom/internal/MyStromHandler.java

```diff
@@ -97,7 +84,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     private @Nullable MyStromReport getReport() {
         try {
             String returnContent = sendHttpGet("report");
-            MyStromReport report = gson.fromJson(returnContent, MyStromReport.class);
+            MyStromReport report = getGson().fromJson(returnContent, MyStromReport.class);
```

Comment:
If you make `gson` in the abstract handler `protected final` instead of `private final` you can access the field directly and remove the getter.

Reply:
I learned this as a best practice a long time ago ;-) I can do it like this... it's also easier.